pipeline/src/resources/ColorAnnotator.py

```diff
@@ -0,0 +1,59 @@
+
+import tornado.web
+import socket
+import tornado.ioloop
+import json
+
+
+class Annotator(tornado.web.RequestHandler):
+    def __init__(self):
+        self._addr = ''
```

Comment:
`self._addr` is never set to anything meaningful. We should probably set it to `'127.0.0.1'`.

Reply:
Addressed in #20

Reply:
@Lewis-Kelley `''` is actually meaningful. It indicates that you should listen to all available interfaces

Reply:
What do you mean by "available interfaces?" Does that mean it can receive requests from anywhere?

Reply:
Anything that can reach the computer I believe. I'm also not certain how that interacts with the OS. There is more info on it here: https://docs.python.org/3/library/socket.html#socket-families